Sources/Logging/Log.swift

```diff
@@ -8,136 +8,137 @@
 
 import Foundation
 
-public final class Log: Logger {
+public enum Log {
 
-    public enum Error: Swift.Error {
+    // MARK: - Default Logger
+
+    public enum DefaultLoggerError: Swift.Error {
         case duplicateDestination(LogDestination.ID)
         case inexistentDestination(LogDestination.ID)
+        case duplicateModule(String)
+        case inexistentModule(String)
     }
 
-    /// Queue object used to specify `DispatchQueue`'s used in log destinations, ensuring they are serial queues of the
-    /// specified QoS, targeting an optional queue.
-    public final class Queue {
-        public let dispatchQueue: DispatchQueue
+    public final class DefaultLogger<Module: LogModule>: Logger {
 
-        public init(label: String, qos: DispatchQoS = .background, target: DispatchQueue? = nil) {
-            dispatchQueue = DispatchQueue(label: label, qos: qos, target: target)
-        }
-    }
+        public private(set) var destinations = Atomic<[LogDestination]>([])
+        public private(set) var modules = Atomic<[Module : Log.Level]>([:])
 
-    public private(set) var destinations = Atomic<[LogDestination]>([])
-    public var errorClosure: ((LogDestination, Item, Swift.Error) -> ())? = { destination, item, error in
-        print("💥[Alicerce.Log]: Failed to log item \(item) to destination '\(destination.id)' with error: \(error)")
-    }
+        public var errorClosure: ((LogDestination, Swift.Error) -> ())? = { destination, error in
```

Comment:
💄 `onError` reads better than `errorClosure`. The type already tells us it's expecting a closure, while the name itself tells us it's a handler called on situations of failure.

Reply:
Well spotted! Completely agree 😉